src/components/Button/index.tsx

```diff
@@ -20,6 +20,12 @@ export const ButtonSize = Enum({
 });
 export type ButtonSize = Enum<typeof ButtonSize>;
 
+export const IconPosition = Enum({
```

Comment:
Do you want to use string literal types for simple options like `right|left`? We have them in a few other places:
https://github.com/Microsoft/YamUI/blob/master/src/components/Block/index.tsx#L35
https://github.com/Microsoft/YamUI/blob/master/src/components/HorizontalList/horizontal-list.tsx#L9

Reply:
Consistency is good, and I think I like this better anyway. I'l make this change after I get CI to pass.